packages/orchestrator-ui-components/src/pages/metadata/WFOProductBlocksPage.tsx

```diff
@@ -137,12 +138,16 @@ export const WFOProductBlocksPage = () => {
         createdAt: {
             field: PRODUCT_BLOCK_FIELD_CREATED_AT,
             name: t('createdAt'),
-            render: parseDateToLocaleString,
+            render: (date) => <WFODateTime dateOrIsoString={date} />,
+            renderDetails: parseIsoString(parseDateToLocaleDateTimeString),
+            clipboardText: parseIsoString(parseDateToLocaleDateTimeString),
         },
         endDate: {
             field: PRODUCT_BLOCK_FIELD_END_DATE,
```

Comment:
Bit of code duplication in the handling of createdAt/endDate (in multiple locations)

Reply:
This is the config object for the table, therefore we have to tell the table how to render a specific field in case it is an object , like this Date here